zookeeper/server/init.sls

```diff
@@ -69,6 +69,8 @@ zookeeper-service:
     - template: jinja
     - context:
       alt_home: {{ zk.alt_home }}
+      user: {{ zk.user }}
+      group: {{ zk.group }}
```

Comment:
I guess the systemd unit file should not be owned by Zookeeper user as well.

Reply:
Hi @vutny,
line 72/73 does not control owner from systemd unit file itself. Here is user and group passed to zookeeper.service.j2 jinja file. This controls which user is used to start the service.

Reply:
Whoops, my fault. Was looking to the wrong place. Sorry.